src/quilla/hookspecs/steps.py

```diff
@@ -52,3 +55,62 @@ def quilla_context_obj(ctx: Context, root: str, path: Tuple[str]) -> Optional[st
     Returns
         the data stored at the context object if existing, None otherwise
     '''
+
+
+@hookspec(firstresult=True)
+def quilla_store_image(
+    ctx: Context,
+    baseline_id: str,
+    image_bytes: bytes,
```

Comment:
bytes are how the image screenshot is given by Selenium, and it is the easiest way to send it back and forth since it can easily be stored as a file by opening the file in binary form, or loaded into an Image object by going through the BytesIO like the VisualParity object does